crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -67,48 +67,40 @@ pub enum State {
     Error,
 }
 
-pub struct SyncService {
-    /// Room list service used to synchronize the rooms state.
```

Comment:
Why did you remove all the fields comments? :/

Reply:
Oh, one of them was lost in the move. Or do you mean all those should be replicated in the `Inner` struct?

Anyways, I added the missing one back: 9ae02355a8041558b5ea19156e1536f6d0e35599.